docs/historic.md

```diff
@@ -30,4 +30,24 @@ res.to_pandas().head()
 | 2022-06-01 07:00:00+00:00 |         13 |    62 |    12 |
 | 2022-06-01 07:30:00+00:00 |         13 |     0 |     0 |
 | 2022-06-01 08:00:00+00:00 |         12 |     0 |     0 |
-| 2022-06-01 08:30:00+00:00 |         12 |     0 |     0 |
\ No newline at end of file
+| 2022-06-01 08:30:00+00:00 |         12 |     0 |     0 |
+
+
+### Example of multi period request
+
+```python
+from solcast import historic
+import pandas as pd
+
+start_str = '2020-01-01'
+start = pd.Timestamp(start_str)
+days_required = 356
+days = 0
+
+while days < days_required:
+    startdate = start + pd.DateOffset(days=days)
+    startdate_str = startdate.strftime('%Y-%m-%d')
```

Comment:
don't need to format as a string - the SDK can take pandas timestamps directly

Reply:
I received an error when trying to use the datetime object. And the final value doesn't include the 'T' for ISO, which may go through, but this allows me to be explicit:
startdate:  2020-01-01 00:00:00+00:00
startdate_str:  2020-01-01T00:00:00.000Z

With updated line 13: startdate_str = startdate.strftime('%Y-%m-%dT00:00:00.000Z')